package.json

```diff
@@ -38,7 +38,9 @@
     "@storybook/addon-actions": "6.5.14",
     "@storybook/addon-essentials": "6.5.14",
     "@storybook/addon-links": "6.5.14",
+    "@storybook/builder-webpack5": "^6.5.14",
```

Comment:
Can we pin these? (i.e. remove the `^` prefix)

Reply:
Done